app/assets/javascripts/discourse/tests/fixtures/notification-fixtures.js

```diff
@@ -42,7 +42,6 @@ export default {
       {
         id: 789,
         notification_type: NOTIFICATION_TYPES.group_message_summary,
-        acting_user_avatar_template: "/letter_avatar_proxy/v4/letter/o/f05b48/{size}.png",
```

Comment:
I spoke with mark via chat on why he removed this instance of `acting_user_avatar_template` but left the remaining uses of the property in other fixtures, eg.

<img width="443" alt="Screenshot 2024-02-06 at 1 49 52 PM" src="https://github.com/discourse/discourse/assets/50783505/a8fa9096-0b3a-4d75-a3ee-504d22e75dbb">

This is because removing this instance of `acting_user_avatar_template` triggers the system avatar to be used.